content/contents.lr

```diff
@@ -0,0 +1,6 @@
+_model:
```

Comment:
is there any difference if the put the event next to the model definition?

_model: event

Reply:
Both are valid options because the lektor parser split the text using the substring `---` but it's a good practice put the value in front the key `_model: event`

I'll change this line. Thanks for your comment 👍